spec/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-cloud_volume_spec.rb

```diff
@@ -1,26 +1,26 @@
 describe MiqAeMethodService::MiqAeServiceUser do
-  let(:cloud_volume)         { FactoryBot.create(:cloud_volume_openstack) }
+  let(:cloud_volume)         do
+    FactoryBot.create(:cloud_volume_openstack, :ext_management_system => FactoryBot.create(:ems_openstack))
+  end
   let(:service_cloud_volume) do
     MiqAeMethodService::MiqAeServiceManageIQ_Providers_Openstack_CloudManager_CloudVolume.find(cloud_volume.id)
   end
 
   before do
-    zone = FactoryBot.create(:zone)
-    allow_any_instance_of(CloudVolume).to receive(:my_zone).and_return(zone.name)
```

Comment:
This stub was actually masking a real bug, `CloudVolume#my_zone` doesn't exist so this would fail IRL

Fixed in https://github.com/ManageIQ/manageiq/pull/19815/commits/8123d5f5b2f8b53eb4a4897ad842478a22b75102

Reply:
Fixed now